test/hotspot/jtreg/compiler/lib/ir_framework/IRNode.java

```diff
@@ -1337,6 +1337,16 @@ public class IRNode {
         vectorNode(RSHIFT_VI, "RShiftVI", TYPE_INT);
     }
 
+    public static final String SAFEPOINT_SCALAR_MERGE = PREFIX + "SAFEPOINT_SCALAR_MERGE" + POSTFIX;
+    static {
+        beforeMatchingNameRegex(SAFEPOINT_SCALAR_MERGE, "SafePointScalarMerge");
```

Comment:
nit: Safepoint is the more frequently-used capitalization (although I see both in my searches).